integration/test/app/error.go

```diff
@@ -0,0 +1,23 @@
+// +build k8srequired
+
+package app
+
+import "github.com/giantswarm/microerror"
+
+var notMatching = &microerror.Error{
```

Comment:
This should be `notMatchingError`. Same for `notDeleted`. I would call both of them `testError` because we don't match against them.